### ISSUE TITLE
How to make oh-my-zsh's plugin 'colorize' work? It requires a python 'Pygmentize' package.

### ISSUE THREAD

**Issue by Drugoy** [babun/babun#578]
**Disclaimer: I am a total newb, just installed babun.**

I'm not sure it's babun's issue, so please don't regard this ticket as a bug report, I just need some help figuring out how to use babun and its sub-utils.

By default babun uses oh-my-zsh.

Oh-my-zsh supports plugins, <a href="https://github.com/robbyrussell/oh-my-zsh#enabling-plugins">its docs say</a> I just need to edit ``~/.zshrc`` (by adding more plugins to the ``plugins=(git)`` line there).

So I set it to ``plugins=(git colorize)``, restart babun (for changes to get applied), then I run ``colorize`` and it says ``pachage \'pygmentize\' is not installed!``

I googled what's ``pygmentize`` and found <a href="https://github.com/nishigori/oh-my-zsh-custom/blob/master/local/pygments.zsh.dist">this</a>.
I haven't figured out what to do with the ``pygments.zsh.dist`` file, but from its code I figured out that ``pygmentize`` is not a package, but a name of a wrapping function of ``pycat`` and ``pyless`` from a python package called <a href="https://pypi.python.org/pypi/Pygments">``Pygments``</a>.

As it is a python package - it should be installed from python using ``pip`` package manager, but it doesn't work in babun's python:
```
{ ~ }  » python                                                                                 ~
Python 2.7.8 (default, Jul 28 2014, 01:34:03)
[GCC 4.8.3] on cygwin
Type "help", "copyright", "credits" or "license" for more information.
>>> pip
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'pip' is not defined
```

even after I set ``plugins=(git python pip colorize)`` in ``~/.zshrc``.

Is it a bug that ``pip`` doesn't work? Both python and pip plugins exist in the ``\.babun\cygwin\usr\local\etc\babun\home\oh-my-zsh\.oh-my-zsh\plugins`` folder.

To sum it up: could you, please, describe detailed steps to make colorize work in babun?

**Comment by Fleshgrinder** [comment]
The Python package manager `pip` is itself an executable:

```sh
$ pip --help
$ pip install pygments
```

**Comment by Drugoy** [comment]
I get ``zsh: command not found: pip``.

**Comment by vladware** [comment]
Try `pact find pygments`

**Comment by Drugoy** [comment]
@vladware thank you!
